mod/01_rating/set/type/metric_value.rb

```diff
@@ -25,32 +25,34 @@ def source_exist?
   text_card = subfield :text
   link_card = subfield :wikirate_link
 
-  ( file_card && file_card.attachment.present? ) || 
-  ( text_card && text_card.content.present? ) ||
-  ( link_card && link_card.content.present? )
-
+  (file_card && file_card.attachment.present?) ||
+    (text_card && text_card.content.present?) ||
+    (link_card && link_card.content.present?)
 end
 
-event :set_metric_value_name, :before=>:set_autoname, :when=>proc{|c| c.cardname.parts.size < 4} do
+event :set_metric_value_name,
+      before: :set_autoname, when: proc { |c| c.cardname.parts.size < 4 } do
   self.name = ['metric', 'company', 'year'].map do |name|
-      content = remove_subfield(name).content
-      content.gsub('[[','').gsub(']]','')
-    end.join '+'
+    content = remove_subfield(name).content
+    content.gsub('[[', '').gsub(']]', '')
+  end.join '+'
 end
 
-event :create_source_for_metric_value, :before=>:process_subcards, :on=>:create do
+event :create_source_for_metric_value, before: :process_subcards, on: :create do
   create_source
 end
 
-event :create_source_for_updating_metric_value, :before=>:process_subcards, :on=>:update, :when=>proc{  |c| c.source_exist? } do
+event :create_source_for_updating_metric_value,
+      before: :process_subcards,
+      on: :update, when: proc {  |c| c.source_exist? } do
   create_source
 end
 
 def clone_subcards_to_hash subcards
   source_subcards = {}
-  subcards.subcards.each_with_key do |subcard, key|
+  subcards.subcards.each_with_key do |subcard, _key|
     subcard_key = subcard.tag.key
-    if key == 'file'
+    if subcard_key == 'file'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.